receiver/mysqlreceiver/metadata.yaml

```diff
@@ -283,6 +287,16 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [threads]
+  mysql.bytes.count:
+    enabled: true
+    description: The number of transmitted bytes.
```

Comment:
What kind of transmission is this? Network? To/from disk? I think this should be clear in the description and probably reflected in the name too.

Reply:
This is client transmission, so network

Reply:
👍 I think it should end with `network.io` then, maybe `mysql.client.network.io`?

Reply:
changed to `mysql.client.network.io.count`, not sure if `count` should be applied here, I can strip it

Reply:
Since the semantic conventions specifically call out `.io` metrics, I think `count` can be omitted.

Reply:
removed and rebased